src/test/ui/extern/issue-64655-extern-rust-must-allow-unwind.rs

```diff
@@ -46,6 +46,8 @@
 //[thin1]compile-flags: -C opt-level=1 -C lto=thin
 //[thin2]compile-flags: -C opt-level=2 -C lto=thin
 //[thin3]compile-flags: -C opt-level=3 -C lto=thin
+// FIXME(#81825) running this revision without optimizations triggers an llvm assert
```

Comment:
Did you mean to use the number of this PR?

Reply:
Yes; should I create a new issue instead?

Reply:
If you're planning to merge the PR with the FIXME, then yes.

Reply:
Maybe we can broaden the scope of #83854 instead? Various related constallations trigger the same LLVM assert.

Reply:
I created a new issue for now but I guess the two could also be consolidated.